git-reauthor: Add page (#6651)
tldr-pages/tldr: pages/common/git-reauthor.md

# git reauthor

> > Replace details about an author identity, since this command rewrites git's history --force will be needed when `git push` is used.

Reviewer: ```suggestion
> Change details about an author identity. Since this command rewrites the Git history, `--force` will be needed when pushing next time.
```

> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-reauthor>.

> - Change the email and name:

Reviewer: Does this change all occurences of that author? If so, I'd write
```suggestion
- Change an author's email and name across the whole Git repository:
```

Reviewer 2: Yes

`git reauthor --old-email {{old@example.com}} --correct-email {{new@example.com}} --correct-name "{{name}}"`

- Change the email and name to the ones defined in the Git config:

`git reauthor --old-email {{old@example.com}} --use-config`

> - Change the email and name of all commits:

Reviewer: So this changes every single commit to one single author? How about this, to be more clear:
```suggestion
- Change the email and name of all commits, regardless of their original author:
```

Reviewer 2: lgtm

# git reauthor

> Replace details about an author identity, since this command rewrites git's history --force will be needed when `git push` is used.
> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-reauthor>.

- Replace the personal email and name of Jack to his work ones:

> `git reauthor --old-email jack@perso.me --correct-email jack@work.com --correct-name 'Jack Foobar'`

Reviewer 3: Are you sure, you have read our style guide?

```suggestion
`git reauthor --old-email {{old@example.com}} --correct-email {{new@example.com}} --correct-name "{{name}}"`
```

Author: Sorry about that, my bad
